controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -3323,6 +3324,103 @@ bool master_thread::handle_cmdu_control_message(const std::string &src_mac,
 #endif
         break;
     }
+    case beerocks_message::ACTION_CONTROL_CHANNEL_SCAN_TRIGGER_SCAN_RESPONSE: {
+        LOG(TRACE) << "ACTION_CONTROL_CHANNEL_SCAN_TRIGGER_SCAN_RESPONSE for mac " << hostap_mac;
+        break;
+    }
+    case beerocks_message::ACTION_CONTROL_CHANNEL_SCAN_TRIGGERED_NOTIFICATION: {
+        LOG(TRACE) << "DCS_task, sending SCAN_TRIGGERED for mac " << hostap_mac;
+        auto notification =
+            beerocks_header->addClass<beerocks_message::cACTION_CONTROL_CHANNEL_SCAN_TRIGGERED_NOTIFICATION>();
+        if (notification == nullptr) {
+            LOG(ERROR) << "addClass cACTION_CONTROL_CHANNEL_SCAN_TRIGGERED_NOTIFICATION failed";
+            return false;
+        }
+
+        auto radio_mac = notification->radio_mac();
+
+        dynamic_channel_selection_task::sScanEvent new_event;
+        new_event.radio_mac = radio_mac;
+        tasks.push_event(database.get_dynamic_channel_selection_task_id(radio_mac),
```

Comment:
What if there's no DCS task for this MAC?

Reply:
Added a check and an error log in case it doesn't find one